job_bundles/afterfx_render_one_task/template.yaml

```diff
@@ -73,15 +82,16 @@ steps:
   script:
     actions:
       onRun:
-        command: aerender
-        args:
-        - "-project"
-        - "{{Param.ProjectFile}}"
-        - "-comp"
-        - "{{Param.CompName}}"
-        - "-output"
-        - "{{Param.OutputDirectory}}"
-        - "-s"
-        - "{{Param.StartFrame}}"
-        - "-e"
-        - "{{Param.EndFrame}}"
+        command: bash
+        args: ['{{Task.File.Run}}']
+    embeddedFiles:
+    - name: Run
```

Comment:
nit: I'd recommend adding a file extension to the name that gets created. Makes it easier to inspect for debugging purposes. ie. your file editor will recognize the syntax for the file extension